python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -534,12 +537,15 @@ def getTableConstraints(self, table):
         schema, tablename = self.getSchemaTableName(table)
         schema_where = u" AND nspname=%s " % self.quoteString(schema) if schema is not None else ""
 
-        sql = u"""SELECT c.conname, c.contype, c.condeferrable, c.condeferred, array_to_string(c.conkey, ' '), c.consrc,
+        version_number = int(self.getInfo()[0].split(' ')[1].split('.')[0])
+        con_col_name = 'consrc' if version_number < 12 else 'conbin'
```

Comment:
I think you can put `<11 here`

Reply:
I've not tested 11, I only have 8, 9, 10 and 12 here. Are you sure that 11 does not use `consrc` too? Looking here it looks like it does: https://www.postgresql.org/docs/11/catalog-pg-constraint.html

Reply:
hello, i'm fairly new to GitHub however i'm encountering the same problem.  What does 'postgis 12 adscr -> adbin consrc -> conbin mean as a solution'?